test/tst_dyn_mv_history.jl

```diff
@@ -63,10 +63,35 @@ end
 
     a1, a2 = get(_history, :mystring)
     @test typeof(a1) <: Vector{UInt8}
-    @test typeof(a2) <: Vector{ASCIIString}
+    @test typeof(a2) <: Vector{String}
 
     a1, a2 = get(_history, :myfloat)
     @test typeof(a1) <: Vector{UInt8}
     @test typeof(a2) <: Vector{Float32}
 end
 
+
+@testset "DynMultivalueHistory: @trace" begin
+    _history = DynMultivalueHistory()
+    n = 2
+    x = linspace(0,1,n)
+    for i = 1:n
+        xi = x[i]
+        @test @trace(_history, i, xi) == xi
+        @test @trace(_history, i, round(Int,xi)) == round(Int,xi)
```

Comment:
can you collapse those two lines into one? So that we actually test that this macro can handle more than one variable at a time